saleor/graphql/core/connection.py

```diff
@@ -1,11 +1,228 @@
-from typing import Union
+import typing
```

Comment:
Why not import the ones we want? `foo: typing.List[typing.Any]` is not very readable.

Reply:
`from graphene import Field, List, NonNull, ObjectType, String`
`from typing import List, Union`

Reply:
In this case, I think it's more readable to import `graphene` as a namespace.

Reply:
done